src/lib/constants.ts

```diff
@@ -42,6 +42,21 @@ export const CONFIG_EXPERT: MinesweeperConfig = { x: 30, y: 16, mines: 99 };
  */
 export const CONFIG_DEFAULT: MinesweeperConfig = { x: 0, y: 0, mines: 0 };
 
+/**
+ * Enum respresenting the overall game state.
+ */
+export enum GAME_STATE {
+  /** The default state of a board before it has been seeded with mines. */
+  DEFAULT,
+  /** The state of a board after it has been seeded with mines. */
+  SEEDED,
+  /** A state representing the player having won. B) */
+  WIN,
+  /** A state representing the player having lost. X( */
+  LOSE
```

Comment:
Whoops! This should come before `WIN`.

Reply:
This matters because the values of this `enum` are interpreted as ascending numbers. There is other logic introduced in these changes that assume the `LOSE` has a lower value than `WIN`.